app/assets/components/ui-elements/oj-review-paper-info.html

```diff
@@ -129,6 +129,7 @@
               font-weight: 700;
               /*line-height: 30px;/* Approximation due to font substitution */
               max-height:  92px;
+              overflow-y:  hidden;
```

Comment:
I think this is probably better than `scroll` but would it be possible to instead truncate the title? Is there a Polymer helper function to do this?